doc/utils.rst

```diff
@@ -1,17 +0,0 @@
-``gempython.gemplotting.utils`` --- Utilities
-=============================================
-
```

Comment:
my ignorance again, why's this file being removed?

Reply:
The packages previously listed there are included in another way, each on a separate page. It makes it easier to tell the difference.